landbosse/model/CollectionCost.py

```diff
@@ -615,7 +695,16 @@ def estimate_construction_time(self, construction_time_input_data, construction_
 
         throughput_operations = construction_time_input_data['rsmeans']
         trench_length_km = construction_time_output_data['trench_length_km']
-        operation_data = throughput_operations.where(throughput_operations['Module'] == 'Collection').dropna(thresh=4)
+        if construction_time_input_data['turbine_rating_MW'] >= 0.1:
+            operation_data = throughput_operations.where(throughput_operations['Module'] == 'Collection').dropna(
+                thresh=4)
+            # from rsmeans data, only read in Collection related data and filter out the rest:
+            cable_trenching = throughput_operations[throughput_operations.Module == 'Collection']
+        else:   #switch for small DW
+            operation_data = throughput_operations.where(
+                throughput_operations['Module'] == 'Small DW Collection').dropna(thresh=4)
+            # from rsmeans data, only read in Collection related data and filter out the rest:
+            cable_trenching = throughput_operations[throughput_operations.Module == 'Small DW Collection']
```

Comment:
Why are you creating a separate operation type for DW? I don't think we should do that but I'm open to hearing an argument for it.

Reply:
@eberlea that's for ≤ 100 kW turbines (that is, small DW). It has data that's separate from projects with > 100 kW turbines.

Reply:
As below: Yes, thanks for confirming that the data are different. However, just because the calculations use different cost data doesn't mean that they should be reported under a separate name in the output dictionary. I think we should be using consistent naming conventions for output regardless of whether the model is generated results for a distributed run. Does that make sense? @akey7 and @parangat94: please review and revise.